patch/0001-hwmon-ltc2978-Add-support-for-LTC3888.patch

```diff
@@ -0,0 +1,120 @@
+From 7096c2c942eb2294e42b279f6fd60d780fb13db5 Mon Sep 17 00:00:00 2001
+From: Ashwin H <Ashwin.H@amd.com>
+Date: Tue, 27 Jun 2023 06:32:53 +0000
+Subject: [PATCH 01/29] hwmon: (ltc2978) Add support for LTC3888
+
+Add support for LTC3888 and LTC3888-1 driver
+Add LTC3888 IOUT support.
+
```

Comment:
It’d be great if you shared a rough timeline.

Reply:
I oppose committing anything if no such ETA is presented. Otherwise SONiC won’t be able to estimate the maintenance costs. Even with an ETA and vendor promises, I have enough experience, that vendors promised to do one thing, but once their changes were excepted in SONiC and their project was done, they moved on, and maintenance is left to the SONiC community.

Reply:
>> I have enough experience, that vendors promised to do one thing, but once their changes were excepted in SONiC and their project was done, they moved on, and maintenance is left to the SONiC community.

This is what i want to avoid and set the realistic expectation.